exercises/hello-workflow/README.md

```diff
@@ -0,0 +1,74 @@
+# Exercise 1: Hello Workflow
+
+During this exercise, you will
+* Review the business logic of the provided Workflow Definition to understand its behavior
+* Modify the Worker initialization code to specify a task queue name
+* Run the Worker initialization code to start the Worker process
+* Execute the Workflow from the command line, specifying your name as input
+
+Make your changes to the code in the `practice` subdirectory (look for TODO 
+comments that will guide you to where you should make changes to the code). 
+If you need a hint or want to verify your changes, look at the complete version 
+in the `solution` subdirectory.
+
+## Part A: Review the Workflow Business Logic
+
+1. Open the `Workflow.cs` file (located in the `Worker` folder under `practice`) 
+   in the editor
+2. Review the input parameters, business logic, and return value. 
+
+## Part B: Specify a Task Queue Name for the Worker
+
+1. Open the `Program.cs` file (located in the `Client` folder under `practice`) 
+   in the editor
+2. Write `greeting-tasks` as the name of the task queue
+3. Save your changes
+4. Repeat and open the `Program.cs` file in the `Worker` folder under `practice)
+5. Write `greeting-tasks` as the name of the task queue and save
+
+
+## Part C: Start the Worker
+
+1. Make sure you're in the practice subdirectory. Run the following command in the terminal window to start the Worker
```

Comment:
It may be worth providing the command in this exercise and _all_ exercises to have them change into the correct subdirectory. We don't require Linux knowledge to take the course, so we should probably provide these commands.

```
Change into the practice subdirectory using the following command:

```bash
cd /workspace/edu-101-dotnet-code/exercises/hello-workflow/practice/
```

Or you can include aliases `ex1` would do that. Check other courses for these. The Java ones in particular take advantage of aliases.